Automattic/sensei
# Add extensions update banner

### Changes proposed in this Pull Request

* It introduces the update banner functionality. I was not sure about the correct designs because we have some different versions, so I deduced it. cc @sruj09 
* It also updates the banner icon.

### Testing instructions

* Open the extensions page, and make sure the update banner works well.
* Change the `extensionsMock` variable in `assets/extensions/main.js`, adding the props `has_update` and `version` to the items to simulate different scenarios
  * The banner should not appear without any update.
  * Design for a single update.
  * Design for multiple updates.
* It doesn't have the image yet.
* It doesn't have the update functionality yet. 
* The link is pointing to the plugin link. In the future, it'll point to the changelog.


### Screenshot / Video

<img width="1216" alt="Screen Shot 2021-04-08 at 18 12 37" src="https://user-images.githubusercontent.com/876340/114097659-bb4f8800-9896-11eb-909f-6c421968c8c3.png">

<img width="1208" alt="Screen Shot 2021-04-08 at 18 12 23" src="https://user-images.githubusercontent.com/876340/114097670-bdb1e200-9896-11eb-80ce-4532801ca7fb.png">
